docs/src/main/paradox/java/http/common/index.md

```diff
@@ -3,7 +3,7 @@
 HTTP and related specifications define a great number of concepts and functionality that is not specific to either
 HTTP's client- or server-side since they are meaningful on both end of an HTTP connection.
 The documentation for their counterparts in Akka HTTP lives in this section rather than in the ones for the
-@ref[Client-Side API](../client-side/index.md), @ref[Low-Level Server-Side API](../server-side/low-level-server-side-api.md) or @ref[High-level Server-Side API](../routing-dsl/index.md),
+@ref[Client-Side API](../client-side/index.md), @ref[Low-Level Server-Side API](../low-level-server-side-api.md) or @ref[High-level Server-Side API](../routing-dsl/index.md),
```

Comment:
Even if the high-level server side is inside the routing-dsl directory, IMO it is better to have a server-side directory to match the common and client-side directories.